### PR TITLE
Add new cisco asa templates

### PR DESCRIPTION
**Pull request by sjhloco**
Added the following new cisco_asa templates, they pretty much follow same format as the equivalent IOS templates

- show interface ip brief
- show port-channel summary
- show ospf interface brief
- show ospf neighbor

### CODE REVIEW

File: ntc_templates/templates/cisco_asa_show_ospf_neighbor.textfsm [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Value NEIGHBOR_ID (\d+.\d+.\d+.\d+)
```
Comment (itdependsnetworks) [inline ntc_templates/templates/cisco_asa_show_ospf_neighbor.textfsm:1]:
```suggestion
Value NEIGHBOR_ID (\d+\.\d+\.\d+\.\d+)
```

File: ntc_templates/templates/cisco_asa_show_ospf_neighbor.textfsm [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+Value PRIORITY (\d+)
+Value STATE (\S+\/\s+\-|\S+)
+Value DEAD_TIME (\d+:\d+:\d+)
+Value ADDRESS (\d+.\d+.\d+.\d+)
```
Comment (itdependsnetworks) [inline ntc_templates/templates/cisco_asa_show_ospf_neighbor.textfsm:5]:
```suggestion
Value ADDRESS (\d+\.\d+\.\d+\.\d+)
```

File: ntc_templates/templates/cisco_asa_show_interface_ip_brief.textfsm [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Value INTF (\S+)
+Value IPADDR (\S+)
+Value STATUS (up|down|administratively down)
+Value PROTO (up|down)
+
+Start
+  ^${INTF}\s+${IPADDR}\s+\w+\s+\w+\s+${STATUS}\s+${PROTO} -> Record
+
```
Comment (itdependsnetworks) [inline ntc_templates/templates/cisco_asa_show_interface_ip_brief.textfsm:8]: Can you add:

```
  ^\s*$$
  ^. -> Error
```

To all of the templates as well.

Comment (sjhloco) [reply ntc_templates/templates/cisco_asa_show_interface_ip_brief.textfsm]: all done

**Pull request merged**